### PR TITLE
Updating Evaluation Code

### PR DESCRIPTION
**Pull request by griegler**
I updated the evaluation code:

1. Using current version open3d and assume it is installed in the Python path. Therefore, no need for configs in `setup.py`.
2. Moved the other necessary parameters to the command prompt in `run.py`. Again, no more changes in setup.py needed to get the evaluation running. Script is now started for each scene independently.
3. Removed dependency on compiled open3d examples. Implemented the color coding in pure Python.

In general, this should generate the very same output. The only thing that might slightly differ is the color coding, because I am not certain that the `hot` color map as implemented in open3d is the very same as implemented in matplotlib.

### CODE REVIEW

**Comment by griegler** [comment]
@syncle @arknapit

**Comment by syncle** [comment]
Thanks Gernot, I will have a look

2020년 1월 16일 (목) 오후 6:35, Gernot <notifications@github.com>님이 작성:

File: python_toolbox/evaluation/README.md [hunk 1]
```diff
@@ -13,11 +13,9 @@ https://tanksandtemples.org/download/
 
 **Step 2**. Install Open3D. Follow instructions in http://open3d.org/docs/getting_started.html
 
-**Step 3**. Clone this repository and follow instructions in ``setup.py``.
-
-**Step 4**. Run the evaluation script and grab some coffee.
+**Step 3**. Run the evaluation script and grab some coffee.
 ```
-python run.py
+python run.py --dataset-dir C:/TanksAndTemples/evaluation/data/Ignatius --traj-path C:/TanksAndTemples/evaluation/data/Ignatius/Ignatius_COLMAP_SfM.log --ply-path C:/TanksAndTemples/evaluation/data/Ignatius/Ignatius_COLMAP.ply
```
Comment (syncle) [inline python_toolbox/evaluation/README.md]: "C:" is Windows style folder. How about removing "C:"?

File: python_toolbox/evaluation/plot.py [hunk 2]
```diff
@@ -36,53 +36,80 @@
...
 
-	# save figure and display
-	f.savefig(png_name, format='png', bbox_inches='tight')
-	f.savefig(pdf_name, format='pdf', bbox_inches='tight')
-	if show_figure:
-		plt.show()
+    # Put a legend to the right of the current axis
+    ax.legend(loc="center left", bbox_to_anchor=(1, 0.5))
+    plt.setp(ax.get_legend().get_texts(), fontsize=pfontsize)
+    png_name = mvs_outpath + "/PR_{0}_@d_th_0_{1}.png".format(
```
Comment (syncle) [inline python_toolbox/evaluation/plot.py]: I see some Python3 scripts (I am not trying to blame this PR, but I just want to mention). How about saying Python3 in the requirement or readme?

File: python_toolbox/evaluation/plot.py [hunk 3]
```diff
@@ -36,53 +36,80 @@
...
+    fscore,
+    dist_threshold,
+    edges_source,
+    cum_source,
+    edges_target,
+    cum_target,
+    plot_stretch,
+    mvs_outpath,
+    show_figure=False,
+):
```
Comment (syncle) [inline python_toolbox/evaluation/plot.py]: could you add requirements.txt file in the root, so the matplotlib friendly functions should produce the same results with your testing environment?

File: python_toolbox/evaluation/plot.py (hunk 2 above)
Comment (arknapit) [reply python_toolbox/evaluation/plot.py]: Pyhton3 as a requirement is good with me.

**Comment by arknapit** [comment]
I still have no write access btw.

**Comment by arknapit** [comment]
> LGTM with minor comments. @arknapit, could you try to test this PR, and confirm it is ok with you?

I tried it out, works fine. Good to merge with me. Thx @griegler for keeping it updated.

File: python_toolbox/evaluation/README.md (hunk 1 above)
Comment (griegler) [reply python_toolbox/evaluation/README.md]: I think I adapted it from something already existent, but I changed it to something more generic.

File: python_toolbox/evaluation/plot.py (hunk 2 above)
Comment (griegler) [reply python_toolbox/evaluation/plot.py]: I added it as req to the readme

File: python_toolbox/evaluation/plot.py (hunk 3 above)
Comment (griegler) [reply python_toolbox/evaluation/plot.py]: I added a requirements.txt.

**Comment by syncle** [comment]
@arknapit @griegler Thanks for the work!

**Pull request merged**